client/pages/apply.jsx

```diff
@@ -78,11 +78,12 @@ class Apply extends React.Component {
                 type="text"
                 required
                 defaultValue={name}
-                pattern="[A-Za-z0-9 ]{3,50}"
+                pattern="[A-Za-z][A-Za-z0-9_\- ]{2,49}"
                 data-error="Please enter"
               />
               <Form.Control.Feedback type="invalid">
-                Your name needs to be between 3 and 50 characters long.
+                Your name needs to be between 3 and 50 characters long and begin with a letter. Underscores and hyphens
```

Comment:
Also, I would change name to be between 2 and 50. A 2-letter name isn't that uncommon.

Reply:
Can't disagree with that, will change.